### ISSUE TITLE
Guild Emblems

### ISSUE THREAD

**Issue by SamHurne** [Ruhrpottpatriot/GW2.NET#13]
**[Moved from Codeplex: https://gw2dotnet.codeplex.com/workitem/1221]**

We have a way to retrieve RGB color information, but we do not have a way to retrieve guild emblem icons. I know that the wiki has a complete gallery of emblems, but it's not clear if we are able/allowed to use that.

http://wiki.guildwars2.com/wiki/Gallery_of_guild_emblems

Also relevant: https://forum-en.guildwars2.com/forum/community/api/Use-of-Guild-Emblems

What can we do about this?

**Comment by Ruhrpottpatriot** [comment]
I have a small Wiki parser written. I'll have to write some more UnitTests for it, but then I'll upload it as a new branch here.

**Comment by sliekens** [comment]
I don't believe that scraping the wiki is the right way to go. Why not just store the images offline?

**Comment by Ruhrpottpatriot** [comment]
You have to get the images at some point and that is the reason for the parser. After that we should store them offline and requery when a new build gets released.

**Comment by sliekens** [comment]
Oh, I didn't think of it that way. In that case I think it's fine if it stays in a separate repository. It doesn't need to be in this one. Especially if you want to add a wiki for it.

**Comment by Ruhrpottpatriot** [comment]
For now it's in a separate repository. Handling a small at this point is less hassle. I'll move it to the main GW2.NET Solution when it's done. At that point the library will also include the link header parser mentioned in #11.
